src/main/java/org/mars/rover/kata/entrydata/StdinProcessor.java

```diff
@@ -0,0 +1,52 @@
+package org.mars.rover.kata.entrydata;
+
+import org.mars.rover.kata.Direction;
+import org.mars.rover.kata.Position;
+import org.mars.rover.kata.commands.CommandParser;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.Scanner;
+
+public class StdinProcessor implements InputProcessorInterface {
+    CommandParser commandParser;
```

Comment:
W przypadkach klas, które nie są `record`ami, a dostają na wejściu jakieś konieczne pola, IMHO czytelniejsze jest z lombokiem. Usuwasz wtedy boilerplate pisania konstruktora, który jedynie przypisuje wartości, a zamiar jest czytelny: obiekt tej klasy musi na wejściu mieć te pola.

```suggestion
@RequiredArgsConstructor
public class StdinProcessor .... {
    private final CommandParser commandParser;
 }
```

Reply:
Dodane do release/1.0.